### ISSUE TITLE
Add DataFramesMeta.jl version

### ISSUE THREAD

**Issue by pdeffebach** [TerseTears/dplyr-DataFrames.jl-cheatsheet#1]
Would you appreciate a PR to have a version of this cheatsheet for DataFramesMeta.jl? Since it's more dpyr-like than DataFrames.jl. At the end of the day, all the code actually run will be identical to the DataFrames.jl version, since DataFramesMeta.jl just creates DataFrames.jl commands.

**Comment by TerseTears** [comment]
Yes, sure. It would be very much appreciated :-). Last time I used DataFramesMeta.jl though, I had difficulties doing everything I needed without having to use DataFrames.jl commands themselves. Nevertheless, I'm sure things have improved a lot since. 

All you would need to do is to add a `meta_code` field to each item in the `juliacheat.yaml`, similar to the `julia_code` field. Additionally, the `juliacheat.tex` template needs to be modified at the line with `$it.julia_code.example/chomp$` and similar lines for the meta fields be added.

To produce the document, the `build.sh` script can be used and pandoc and xelatex would be required. Spacing will likely not look as good after adding yet another field to each cheatsheet block, but that's something I plan to work on myself, so do not worry about that :-).
